refinery/tool_manager/models.py

```diff
@@ -393,6 +393,10 @@ def update_file_relationships_with_urls(self):
         self.set_tool_launch_config(tool_launch_config)
 
 
+class VisualizationToolError(Exception):
```

Comment:
If we are to have a custom exception, might want to inherit from a less-broad exception class